scripts/dataStore.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
If we go this route we're going to need to consider clearing this cache and long-term effects of storing this data. I do not particularly like the opportunity for human error this introduces, primarily for our time table and the low likelihood of perpetual maintenance of this repository. It seems like high memory-cost if we're not careful. So we should think about this with caution, as you've stated.